console-csharp-connect-sample/GraphClientFactory.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.Graph;
+using Microsoft.Identity.Client;
+using System.Linq;
+using console_csharp_connect_sample.Helpers;
+
+namespace console_csharp_connect_sample
+{
+	// This static class returns a fully constructed 
+	// instance of the GraphServiceClient with the client 
+	// data to be used when authenticating requests to the Graph API
+	public static class GraphClientFactory
+	{
+		private static GraphServiceClient _serviceClient;
```

Comment:
You don't need to hold onto the instance here.  It's a factory, so it just helps create an instance of the GraphServiceClient.  The consumer is responsible for holding onto the instance.